go.mod

```diff
@@ -11,6 +11,7 @@ require (
 	github.com/xdg/stringprep v1.0.0 // indirect
 	go.mongodb.org/atlas v0.5.1-0.20201007214134-b315fe7503d2
 	golang.org/x/text v0.3.3 // indirect
+	golang.org/x/tools v0.0.0-20201116182000-1d699438d2cf // indirect
```

Comment:
[q] was this and the changes to the go.sum file necessary? can you please do `go mod tidy`?

Reply:
Yeah. Looks like I forgot to run it. Sorry. It was fixed 👍